src/systems/sensors/Sensors.cc

```diff
@@ -412,6 +412,17 @@ void Sensors::Configure(const Entity &/*_id*/,
   this->dataPtr->Run();
 }
 
+//////////////////////////////////////////////////
+void Sensors::Update(const UpdateInfo &_info,
```

Comment:
Unfortunately I had to do it in `Update` and not `Configure` because `Sensors::Configure` is called before `ThermalSensor::Configure`, which means we would be looking for resolution / temperature range components before they are created.